sormas-api/src/main/resources/enum.properties

```diff
@@ -952,12 +952,14 @@ TaskType.CASE_MANAGEMENT = case management
 TaskType.CONTACT_FOLLOW_UP = contact follow up
 TaskType.CONTACT_INVESTIGATION = contact investigation
 TaskType.CONTACT_TRACING = contact tracing
+TaskType.CONTINUE_INVESTIGATION = continue the investigation following a change in the signal
 TaskType.DAILY_REPORT_GENERATION = generate daily report
 TaskType.DECONTAMINATION_DISINFECTION_ACTIVITIES = decontamination / disinfection activities
 TaskType.ENVIRONMENTAL_HEALTH_ACTIVITIES = environmental health activities
 TaskType.EVENT_INVESTIGATION = investigate the event
 TaskType.OTHER = other task as described in comments
 TaskType.QUARANTINE_PLACE = quarantine place
+TaskType.REQUEST_OR_ADDITIONAL_INFORMATION = request for an investigation report / request for a summary / additional information to be collected
```

Comment:
Rename to EVENT_REQUEST_ADDITIONAL_INFORMATION and remove the second "request for a" from the text to make it a bit more compact